fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -2358,6 +2356,9 @@ public long getConnectorSinkTargetMaxFileSize() {
     @VarAttr(name = ENABLE_COUNT_STAR_OPTIMIZATION, flag = VariableMgr.INVISIBLE)
     private boolean enableCountStarOptimization = true;
 
+    @VarAttr(name = ENABLE_MIN_MAX_OPTIMIZATION)
+    private boolean enableMinMaxOptimization = true;
```

Comment:
I think default value should be false. I remember that trino won't record min/max by default

Reply:
OK.